qutip/solver/integrator/explicit_rk.pxd

```diff
@@ -27,6 +27,7 @@ cdef class Explicit_RungeKutta:
     cdef readonly int max_numsteps
     cdef readonly bint interpolate
     cdef readonly object method
+    cdef dict butcher_tableau
```

Comment:
Should `butcher_tableau` be readonly too?

Reply:
In Cython `readonly` is about accessibility from Python. So `cdef public butcher_tableau` would add a setter and a getter while `cdef readonly butcher_tableau` would add only a getter. I suggested adding a getter for consistency with the rest of the attributes.